src/components/edit-dl-controller.tsx

```diff
@@ -82,20 +121,28 @@ export const EditDLControllerComponent: FC<EditDLControllerComponentProps> = ({
 			originalMembersRef.current,
 			members
 		);
+		const detailDifference = pickBy(
+			details,
+			(value, key) => originalDetailsRef.current[key as keyof DLDetails] !== value
+		);
 		client
-			.distributionListAction(email, membersToAdd, membersToRemove)
+			.distributionListAction({ email, membersToAdd, membersToRemove, ...detailDifference })
 			.then(() => {
```

Comment:
- [ ] Update original data on save since the board is kept open, so that following discards consider updated data